Authentication/AuthenticationBootstrapper.swift

```diff
@@ -31,7 +34,7 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
         return sessionService.currentUser.value
     }
 
-    /**
+    /*
```

Comment:
Why did you change this? If you remove the second *, the user of this class will not be able to see the description.